cmd/feed-dns/main.go

```diff
@@ -104,4 +116,14 @@ func validateConfig() {
 		log.Error("Must supply r53-hosted-zone")
 		os.Exit(-1)
 	}
+
+	if (internalCNAME != "" || externalCNAME != "") && elbLabelValue != "" {
+		log.Error("Can't supply both ELB label and non-ELB CNAME. Choose one or the other.")
+		os.Exit(-1)
+	}
+
+	if (internalCNAME == "" && externalCNAME != "") || (internalCNAME != "" && externalCNAME == "") {
+		log.Error("Must supply both internal-cname and external-cname if any are to be provided.")
+		os.Exit(-1)
```

Comment:
I think this is too strict. I could imagine someone only wanting to use an external host for example.

Reply:
It might be better to move the validation inside of `dns.New` so we can unit test it.

Reply:
Agreed. You shouldn't have to specify both, they can be optional.